event/src/unvalidated/event.rs

```diff
@@ -384,18 +388,22 @@ impl RawTimeEvent {
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Proof {
-    chain_id: String,
-    root: Cid,
-    tx_hash: Cid,
-    tx_type: String,
+    /// eip-155 CHAIN_ID see https://chainid.network https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
+    pub chain_id: String,
```

Comment:
do these fields actually need to public? I know there's concern about making invalid events without using the builder.

also, this should parse as a `ssi::caip2::ChainID` but probably better to stay a string